Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "oxilangtag"
-version = "0.1.2"
+version = "0.2.0"
```

Comment:
It seems to me we are backward compatible so no need to upgrade to 0.2.

Reply:
Changed to 0.1.3